Add `NotContainItemsAssignableTo` (#2266)
fluentassertions/fluentassertions: docs/_pages/releases.md

### What's new
* Added `Be`, `NotBe` and `BeOneOf` for object comparisons with custom comparer - [#2111](https://github.com/fluentassertions/fluentassertions/pull/2111)
* Added `BeSignedWithPublicKey()` and `BeUnsigned()` for assertions on `Assembly` - [#2207](https://github.com/fluentassertions/fluentassertions/pull/2207)

> * Added `NotContainItemsAssignableTo` for asserting that collection does not contain any item assignable to specific type [#2266](https://github.com/fluentassertions/fluentassertions/pull/2266)

Reviewer: ```suggestion
* Added `NotContainItemsAssignableTo` for asserting that collection does not contain any item assignable to specific type - [#2266](https://github.com/fluentassertions/fluentassertions/pull/2266)
```

> * Added `NotContainItemsAssignableTo` for asserting that collection does not contain any item assignable to specific type - [#2266](https://github.com/fluentassertions/fluentassertions/pull/2266)

Reviewer 2: ```suggestion
* Added `NotContainItemsAssignableTo` for asserting that a collection does not contain any items assignable to specific type - [#2266](https://github.com/fluentassertions/fluentassertions/pull/2266)
```

> * Added `NotContainItemsAssignableTo` for asserting that a collection does not contain any items assignable to specific type - [#2266](https://github.com/fluentassertions/fluentassertions/pull/2266)

Reviewer 2: ```suggestion
* Added `NotContainItemsAssignableTo` for asserting that a collection does not contain any items assignable to a specific type - [#2266](https://github.com/fluentassertions/fluentassertions/pull/2266)
```
